client/src/components/Settings/CalculatorTab.jsx

```diff
@@ -0,0 +1,96 @@
+import { useState } from "react";
+
+const CalculatorTab = () => {
+
+  const [ age, setAge ] = useState(0);
+  const [ gender, setGender ] = useState("");
+  const [ weight, setWeight ] = useState(0);
+  const [ height, setHeight ] = useState(0);
+  const [ bmi, setBMI ] = useState(0);
```

Comment:
- I believe we should change the way we are writing the logic for this:
    - We can just grab values such as `weight` and `height` from the redux store for a given logged in user.
    - From a user's POV: they wouldn't want to add info they already included
    -  If you decide to do this, please just leave the state variables for  `age`, `gender`, as those values don't exist however, in a future story,  we can add support for those fields and change it then.
    
    
- This is an **_optional_** change, however, if you would like to keep this, please add error handling for all of the state variables. Before calculating the BMI
    - you can achieve this by adding error state variables for each input or a universal one. Check out other files such as `LoginForm.jsx` etc.

Reply:
Decided to not involve user data for this feature because the intent is for people to use it as a hypothetical calculation. Or just give them the freedom to fool around with it as they please. Additionally, if they were able to constantly see the number from the calculation it may result in a build up of negative feelings.